src/Umbraco.Web.UI.Client/src/views/prevalueeditors/contenttypepicker.controller.js

```diff
@@ -0,0 +1,54 @@
+function ContentTypePickerController($scope, contentTypeResource, editorService, angularHelper) {
+    var vm = this;
+    vm.loading = false;
+    vm.contentTypes = [];
+    vm.remove = remove;
+    vm.add = add;
+
+    var allContentTypes = null;
+
+    function init() {
+        vm.loading = true;
+        contentTypeResource.getAll().then(function (all) {
```

Comment:
@kjac Beware! Any resource that is not entityResource will require specific permissions. This is why pretty much every single picker should use entityResource since it returns bare minimal information and can be accessed by all users. contentTypeResource will definitely explode if the user doesn't have access to the content type tree.

Reply:
@Shazwazza good point; duly noted. I'll keep it in mind when we solve #5983